docs/variants/versions.md

```diff
@@ -6,29 +6,30 @@ The table below contains an up to date list of all supported platforms,
-| Qemu Q35                | [dasharo](https://github.com/dasharo/coreboot/tree/dasharo)                                           | [dasharo](https://github.com/dasharo/coreboot/tree/dasharo)                 | 2024.02      | [433b7a33ee](https://github.com/dasharo/edk2/tree/433b7a33eeb5ead75546f595e494a2a56e2c6d71) |
+|           Platform            |                                            coreboot branch                                            |                                Dasharo base                                 | EDK2 version |                                          EDK2 rev                                           |
+| ----------------------------- | ----------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------- | ------------ | ------------------------------------------------------------------------------------------- |
+| ASRock Rack SPC741D8-2L2T/BCM | [dasharo](https://github.com/dasharo/coreboot/tree/dasharo)                                           | [dasharo](https://github.com/dasharo/coreboot/tree/dasharo)                 | 2024.02      | [ac25544f92](https://github.com/dasharo/edk2/tree/91a7a092361befde7ddf2cd9915828852aff1195) |
+| NovaCustom NUC BOX            | [dasharo](https://github.com/dasharo/coreboot/tree/dasharo)                                           | [dasharo](https://github.com/dasharo/coreboot/tree/dasharo)                 | 2024.02      | [ac25544f92](https://github.com/dasharo/edk2/tree/ac25544f9267598c51f2cb7f3abd08ae5db1318c) |
+| NovaCustom V540TU             | [#548](https://github.com/Dasharo/coreboot/pull/548)                                                  | [dasharo](https://github.com/dasharo/coreboot/tree/dasharo)                 | 2024.02      | [11b2679614](https://github.com/dasharo/edk2/tree/11b26796145e29b2ceeb5a00b130f81246115159) |
+| NovaCustom V540TNx            | [#548](https://github.com/Dasharo/coreboot/pull/548)                                                  | [dasharo](https://github.com/dasharo/coreboot/tree/dasharo)                 | 2024.02      | [51decd4d1f](https://github.com/dasharo/edk2/tree/51decd4d1fd9d8f64b5eec461e32deb5cc7ae03c) |
+| NovaCustom V560TU             | [#548](https://github.com/Dasharo/coreboot/pull/548)                                                  | [dasharo](https://github.com/dasharo/coreboot/tree/dasharo)                 | 2024.02      | [15e8a08772](https://github.com/dasharo/edk2/tree/15e8a0877230c00029bb6fd0843f7494a8bfd8d0) |
+| NovaCustom V560TNx            | [#548](https://github.com/Dasharo/coreboot/pull/548)                                                  | [dasharo](https://github.com/dasharo/coreboot/tree/dasharo)                 | 2024.02      | [51decd4d1f](https://github.com/dasharo/edk2/tree/51decd4d1fd9d8f64b5eec461e32deb5cc7ae03c) |
```

Comment:
Since you've fixed model names of NovaCustom here, maybe also point to `[dasharo](https://github.com/dasharo/coreboot/tree/dasharo)` instead of a merged coreboot PR?